NDB.Covid19/NDB.Covid19/Locales/so.json

```diff
@@ -222,21 +235,21 @@
   "REGISTER_QUESTIONAIRE_ACCESSIBILITY_RADIO_BUTTON_DATEPICKER_TEXT": "Taabo oo taariikh ka dooro jadwalka.",
   "REGISTER_QUESTIONAIRE_ACCESSIBILITY_CLOSE_BUTTON_TEXT": "Ka bax",
   "REGISTER_QUESTIONAIRE_CHOOSE_DATE_POP_UP": "Dooro taariikh",
-  "REGISTER_QUESTIONAIRE_RECEIPT_LINK": "null",
+  "REGISTER_QUESTIONAIRE_RECEIPT_LINK": null,
```

Comment:
"REGISTER_QUESTIONAIRE_RECEIPT_LINK": "https://www.helsenorge.no/en/coronavirus/quarantine-and-isolation/",

Reply:
^change in both excel sheet and .json